src/coffee/bootstrap-tour.spec.coffee

```diff
@@ -704,6 +704,22 @@ describe 'Bootstrap Tour', ->
     expect($('.popover').hasClass('orphan')).toBe true
     $('.popover').remove()
 
+  it 'should use orphan template to show orphan steps', ->
+    @tour = new Tour
+    step = orphan: '<div class="popover orphan-custom-template"></div>'
+    @tour.addStep step
+    template = @tour._template(step, 0)
+
+    expect($(template).hasClass('orphan-custom-template')).toBe true
+
+  it 'should execute orphan template if it is a function', ->
+    @tour = new Tour
+    step = orphan: -> '<div class="popover orphan-custom-template"></div>'
+    @tour.addStep step
+    template = @tour._template(step, 0)
+
+    expect($(template).hasClass('orphan-custom-template')).toBe true
+
```

Comment:
Please add a test that the orphan template is not used and feel free to merge when done.